assets/ajax-cart.js.liquid

```diff
@@ -0,0 +1,627 @@
+/*============================================================================
```

Comment:
Clean cut away from the old version. If bugs pop up around the ajax cart, we'll instantly know if it's v1 or v2.